.swift-version

```diff
@@ -1 +1 @@
-4.1
\ No newline at end of file
+4.1-DEVELOPMENT-SNAPSHOT-2018-03-08-a
```

Comment:
no. This supposes to be ready for 4.1 release. I know .swift-version is restrictive, but snapshot will outdate like.... in few days.

Reply:
Okay cool - I'll remove that change and close the other PR 👍

Reply:
Updated